sim/src/mechanics/driving.rs

```diff
@@ -248,7 +248,25 @@ impl DrivingSimState {
         transit: &mut TransitSimState,
     ) -> bool {
         match car.state {
-            CarState::Crossing(_, _) => {
+            CarState::Crossing(time_int, dist_int) => {
```

Comment:
Hmm, this is tricky. `Crossing` to `Queued` happens before the car reaches the end of the lane; they might even still be moving, stuck behind a slower vehicle. https://dabreegster.github.io/abstreet/trafficsim/discrete_event.html#cars gives more detail about this.

The delay at an intersection starts being counted here, at `car.state = CarState::Queued { blocked_since: now }`. Maybe we can just show delays at intersections and leave it a bit unspecified why the delay happened -- because of congestion upstream in the target lane, because of slow signal timing, because of a long queue of vehicles, because of a single slow vehicle in front of faster ones, etc.

But for this PR, I think it's fine to keep measuring it this way; we'll iterate on it next.